source/_integrations/bosch_alarm.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: Bosch Alarm
+description: Integrate Bosch Alarms.
+ha_category:
+  - Alarm
+ha_release: 2025.3
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@mag1024'
+  - '@sanjay900'
+ha_domain: bosch_alarm
+ha_platforms:
+  - alarm_control_panel
+ha_zeroconf: false
+ha_integration_type: integration
+---
+
+The **Bosch Alarm Panel** {% term integration %} allows you to connect your [Bosch Alarm Panel](https://www.boschsecurity.com) to Home Assistant to control and monitor your Bosch Alarm Panel.
+
+{% include integrations/config_flow.md %}
+
+## Supported devices
+
+- _Solution 2000/3000/4000_
+- B Series: _B3512/B4512/B5512/B6512_
+- G Series: _B8512G/B9512G_
+- _AMAX 2100/3000/4000_
+- _D7412GV4/D9412GV4_ [^1]
+
+[^1]: Firmware 2.0+
+
+## Provided entities
+
+The following {% term entities %} are provided:
+
+- [AlarmControlPanel](#alarm-control-panel)
+
+### Alarm Control Panel
+
+This integration adds an Alarm Control Panel device for each configured area, with the ability to issue arm/disarm commands.
+This entity reports state (_disarmed_, _armed_away_, etc.), and contains custom attributes _ready_to_arm_ (_no_|_home_|_away_), and a _faulted_points_ counter.
```

Comment:
no counter?

Reply:
Ah yep, removed that as i was told to expose those as sensors instead of attributes, so ill add them in a different pull later.